### ISSUE TITLE
QTextEx - question

### ISSUE THREAD

**Issue by smaragdus** [medo64/QTextEx#1]
@medo64 

I have been using [QText](https://www.medo64.com/qtext/) for quite some time (since version 2.32) and I am curious- what is [QTextEx](https://github.com/medo64/QTextEx)- [QText](https://github.com/medo64/QText) rewritten on C++? Does it have more feature? Would **QTextEx** replace **QText** in the future or the two programs would be developed independently? Are there any compiled binaries for testing?

Regards

**Comment by medo64** [comment]
Hi,

QTextEx is probably what's going to be QText 5.00 once I am done with it. However, it's really early in development and while basic functions work there is a lot of things that don't. For example, no rich text support, no file change detection, no proper settings GUI, doesn't play nice with DropBox, etc.

I plan to add missing features eventually but that highly depends on my free time and there is not enough of it to go around :(.

If you are interested in using QText on Linux computer, you can download it from [my Linux repository](https://medo64.com/repositories/deb/dists/stable/main/binary-amd64/qtext_0.1.3_amd64.deb). If you have Windows I would recommend staying with QText for now and I can keep you in mind for eventual beta test.

Thanks for interest!

**Comment by smaragdus** [comment]
@medo64 

Thank you for the detailed response. I didn't know that you also code in C++. I discovered this new project by chance after I updated QText from version 4.52 to version 4.53 a few days ago. This reminds me that for quite some time I plan to report a GUI bug in QText (still present in version 4.53) but it will require to record a short video or to make an animated image to better illustrate what actually happens and I am lazy about doing such stuff. I hope next week I will finally create an image or a video and I will report it in [QText issues](https://github.com/medo64/QText/issues). For the time being I don't have Linux installed mostly for 2 reasons- I don't have enough time and I am a novice in Linux. I am also a bit frightened by UEFI as I have never had dual boot system under Windows 8, once I had it under XP. I didn't know that [QText](https://www.medo64.com/qtext/) could work under Linux either, thanks for the info, this may be helpful for me in the future. And yes, when there are are ready working Windows builds of [QTextEx](https://github.com/medo64/QTextEx) I may do some testing. I also upgraded to [Summae](https://www.medo64.com/summae/) version 1.41- thank you for Uppercase hash check-box in Options. Since this issue was just a question I will close it.

Best regards and happy coding!
